LDAR_Sim/src/programs/method.py

```diff
@@ -374,42 +377,51 @@ def survey_site(
             # TODO determine reasonable fraction of site that can be surveyed to make it worth going
             elif crew.day_time_remaining > (2 * site_travel_time):
+                # Log survey start date
+                if not survey_report.survey_in_progress:
+                    survey_report.survey_start_date = curr_date
+                    survey_report.method = self._name
                 # Mark the survey as in progress
                 survey_report.survey_in_progress = True
                 last_site_survey = True
                 # Log Survey and travel time
                 survey_report.time_spent_to_travel += site_travel_time
-                survey_report.time_surveyed += crew.day_time_remaining - (site_travel_time * 2)
+                survey_report.time_surveyed_current_day = crew.day_time_remaining - (
+                    site_travel_time * 2
+                )
+                survey_report.time_surveyed += survey_report.time_surveyed_current_day
                 crew.day_time_remaining = site_travel_time
-                # Log survey start date
-                survey_report.survey_start_date = curr_date
             # Not enough time left to travel to site
             else:
+                survey_report.time_surveyed_current_day = 0
+                survey_report.time_spent_to_travel = 0
                 site_travel_time = 0
                 last_site_survey = True
         return survey_report, site_travel_time, last_site_survey, site_visit
 
     def _determine_if_site_survey_can_be_completed(
         self,
         survey_report: SiteSurveyReport,
-        site_survey_time: float,
-        site_travel_time: float,
-        crew_time_remaining: float,
+        site_survey_time: int,
+        site_travel_time: int,
+        crew_time_remaining: int,
     ) -> bool:
         # Account for the possibility of the crew needing to return when considering if the
         # Site can be surveyed in a day.
         # TODO review travel home time logic
-        survey_required_time: float = site_survey_time + (site_travel_time * 2)
-        actual_required_time: float = survey_required_time - survey_report.time_surveyed
+        survey_required_time: int = site_survey_time + (site_travel_time * 2)
+        actual_required_time: int = survey_required_time - survey_report.time_surveyed
         survey_can_be_completed: bool = crew_time_remaining >= actual_required_time
 
         return survey_can_be_completed
 
-    def _get_travel_time(self) -> float:
-        if isinstance(self._travel_times, (int, float)):
+    def _get_travel_time(self) -> int:
+        if isinstance(self._travel_times, int):
             return self._travel_times
+        elif isinstance(self._travel_times, float):
+            return round(self._travel_times)
         elif isinstance(self._travel_times, list):
-            return choice(self._travel_times)
+            return round(choice(self._travel_times))
```

Comment:
Nit: I think this should be ceiling instead of round, as we're doing with time, and if you have a fraction of a minute, I would use the next integer. It's more conservative but I think better practice.
An example of where this might be relevant is if an equipment took 0.4 minutes to survey. We would want it to say 1 minute to survey.

Reply:
See my comment above